ReactiveCocoa/NSObject+Intercepting.swift

```diff
@@ -1,9 +1,40 @@
 import Foundation
 import ReactiveSwift
 import enum Result.NoError
+import ReactiveCocoaPrivate
 
 extension NSObject {
 	public func signal(for selector: Selector) -> Signal<(), NoError> {
```

Comment:
I have made progress on this. Do you think the arguments should be bridged as `[Any]`?